src/_guides/language/index.md

```diff
@@ -31,6 +31,10 @@ These two resources are popular with both beginning Dartisans and experts.
 : A list of common questions for those who are interested in stronger
   static typing.
 
+[Sound Dart: Common Problems](/guides/language/common-prob)
```

Comment:
I just noticed that this page's name has a colon, and the FAQ page's name doesn't. Add a colon to the FAQ page name?